platform/ios/app_delegate.mm

```diff
@@ -92,8 +92,28 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 	mainViewController = viewController;
 
-	// prevent to stop music in another background app
-	[[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryAmbient error:nil];
+	NSString *categoryString = GLOBAL_GET("audio/general/ios/session_category");
```

Comment:
```suggestion
	String categoryString = GLOBAL_GET("audio/general/ios/session_category");
```
The conversion to a pointer of `NSString` isn't possible, and it is only ever used as a plain `String` here as far as I can tell

Reply:
String comparisons will work fine yes